### PR TITLE
Create action arguments fields for preference-experiment action

### LINKED ISSUE #58: Add action arguments fields for preference-experiment action
Depends on #54 

Existing UI in DC:
image

### PR DESCRIPTION
**Pull request by rehandalal**
Fixes #58 

r?

### CODE REVIEW

File: extension/content/components/recipes/form/arguments/PreferenceExperimentArguments.js [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+      },
+    });
+  };
+
+  return (
+    <FormGroup>
+      <ControlLabel>{label}</ControlLabel>
+      <div className="d-flex">
+        <span className="pr-2 pt-1">
+          <Toggle value={data.arguments[name]} onChange={handleChange} />
```
Comment (tiftran) [inline extension/content/components/recipes/form/arguments/PreferenceExperimentArguments.js]: `checked={data.arguments[name]}`?

File: extension/content/components/recipes/form/arguments/PreferenceExperimentArguments.js [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+      data: {
+        ...data,
+        arguments: {
+          ...data.arguments,
+          branches: branches.map((b) => {
+            if (preferenceType === "boolean") {
+              return { ...b, value: false };
+            }
+
+            return { ...b, value: "" };
```
Comment (tiftran) [inline extension/content/components/recipes/form/arguments/PreferenceExperimentArguments.js]: Would this override displaying the actual branch values for recipes in the edit and import workflow?

Comment (rehandalal) [reply extension/content/components/recipes/form/arguments/PreferenceExperimentArguments.js]: good catch, i'll fix this

File: extension/content/components/recipes/form/arguments/PreferenceExperimentArguments.js (hunk 1 above)
Comment (rehandalal) [reply extension/content/components/recipes/form/arguments/PreferenceExperimentArguments.js]: oops!

**Review by tiftran — APPROVED** [review]

**Pull request merged**